charts/ccd-definition-store-api/values.preview.template.yaml

```
@@ -8,7 +8,20 @@ java:
   keyVaults:
     ccd:
       secrets:
-        - ccd-definition-s2s-secret
+        - name: definition-store-api-POSTGRES-USER
+          alias: DEFINITION_STORE_DB_USERNAME
+        - name: definition-store-api-POSTGRES-PASS
+          alias: DEFINITION_STORE_DB_PASSWORD
+        - name: ccd-definition-s2s-secret
+          alias: DEFINITION_STORE_IDAM_KEY
+        - name: ccd-ELASTIC-SEARCH-URL
+          alias: ELASTIC_SEARCH_HOST
+        - name: ccd-ELASTIC-SEARCH-PASSWORD
+          alias: ELASTIC_SEARCH_PASSWORD
+        - name: storage-account-primary-connection-string
+          alias: AZURE_STORAGE_CONNECTION_STRING
+        - name: AppInsightsInstrumentationKey
+          alias: azure.application-insights.instrumentation-key
```

```suggestion
        - name: ccd-definition-s2s-secret
          alias: DEFINITION_STORE_IDAM_KEY
```
For preview, we don't actually want _all_ the same details - the reason this previously only had `ccd-definition-s2s-secret` defined was because the other important ones are overridden in the `environment` below.